### DISCUSSION TITLE
Adding support for displays on watermeter

### DISCUSSION THREAD

**Discussion by oversponged** [jomjol/AI-on-the-edge-device#2185] [General]
### The Feature

Many houses in germany have these watermeters. Maybe it is possiple to select first full digital/analog and at the config of digital you can add 3 digits after the decimal and remove the analog.

![watermeter](https://user-images.githubusercontent.com/72872507/224982322-63bb6d5c-c181-483c-af07-35c2e41653a7.jpg)

**Comment by rainerhgit** [comment]
Hi. I'm using such a Watermeter  Sensus iPerl. 
For configuration you can use the digits after the comma with setting in the Postprocessing section: Decimal shift -3
In my config are 6 digits set. Analog section is disabled. This works. 
My main issue is the reflection from the display and the contrast.

**Reply by caco3** [reply]
yep, that is the way to do it.

**Comment by oversponged** [comment]
Hi rainerhgit,
thanks for this solution. I will try this out.
Do you have any very special configuration for the reflection and the contrast?

**Reply by rainerhgit** [reply]
Hi Oversponged.

I have no special configuration.
Tested with some cases around the camera and the watermeter but this was not better than without case.
For now the camera is positioned 4-5cm distance to the watermeter display.
The readout works but i'm trying to reduce the reflections.
There is no case around the configuration and it works in the dark room.
Will try the external LED mode.

Second device read the gasmeter nearly perfect because there are not so much reflections and it is no LCD.
(Gasmeter is a Kromschröder BK-G4)

Image from my watermeter via Digitizer:
<img width="350" alt="image" src="https://user-images.githubusercontent.com/53999765/225954377-ed843b0b-bb2b-4c6d-9ac9-9b7a6f2a701d.png">

<img width="696" alt="image" src="https://user-images.githubusercontent.com/53999765/225956542-ecfa00df-658e-4bad-83bf-f2a6dbb13d41.png">

**Comment by caco3** [comment]
Since the digits are tilted, you could try if the recognition can be improved by rotating the image slightly. The ROIs would then not be on the same X-Axis but that is ok.
